src/types/organization.ts

```diff
@@ -0,0 +1,71 @@
+import { Organization, OrganizationScope } from "../models/Organization";
+import { IIpaPublicAdministration } from "./PublicAdministration";
+
+export interface ISearchedOrganization {
+  fiscalCode: string;
+  ipaCode: string;
+  name: string;
+  legalRepresentative: {
+    familyName: string;
+    firstName: string;
+    fiscalCode: string | null;
+    phoneNumber: string | null;
+  };
+  pecs: ReadonlyArray<string>;
+  scope: OrganizationScope | null;
+  selectedPec?: string;
```

Comment:
this should be an index in the `pecs` array or else you may have a `selectedPec` that is not in `pecs`